lib/lotus/validations/error.rb

```diff
@@ -41,9 +58,10 @@ class Error
       #
       # @since 0.1.0
       # @api private
-      def initialize(attribute, validation, expected, actual)
-        @attribute, @validation, @expected, @actual =
-          attribute, validation, expected, actual
+      def initialize(attribute_name, validation, expected, actual, namespace = nil)
+        @attribute_name, @validation, @expected, @actual, @namespace =
```

Comment:
how about instance variables in many lines separately? According to benchmark by Pete Yandel, 1 line assignment is much slower